<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chatWindow</class>
 <widget class="QMainWindow" name="chatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bithub</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>862</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#groupBox{border:none;border-top-left-radius:10px;border-top-right-radius:10px;border-bottom-left-radius:10px; border-bottom-right-radius:10px;}



</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QListWidget" name="messageWindow">
     <property name="geometry">
      <rect>
       <x>219</x>
       <y>39</y>
       <width>631</width>
       <height>401</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(235, 235, 237);
border-top-left-radius:0px;

border-top-right-radius:0px;



border-bottom-left-radius:10px;

border-bottom-right-radius:10px;


margin-left:10px</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="autoScrollMargin">
      <number>10</number>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>211</width>
       <height>571</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-bottom-right-radius:10px;
color: rgb(255, 255, 255);


border-bottom-left-radius:10px;

border-top-right-radius:10px;

border-top-left-radius:10px;


background-color: rgba(184, 194, 183, 180);</string>
     </property>
    </widget>
    <widget class="QLabel" name="title">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>0</y>
       <width>621</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-top-left-radius:10px;

border-top-right-radius:10px;

border-bottom-left-radius:0px;

border-bottom-right-radius:0px;
background-color: rgb(200, 180, 155, 250);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>公共聊天室</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>450</y>
       <width>621</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(198, 200, 215);
border-top-left-radius:10px;

border-top-right-radius:10px;

border-bottom-left-radius:10px;

border-bottom-right-radius:10px;

</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>601</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(255, 255, 255, 90);

border-top-left-radius:10px;

border-top-right-radius:10px;

border-bottom-left-radius:10px;

border-bottom-right-radius:10px;

border:0px solid rgb(0, 0, 0);</string>
      </property>
      <property name="placeholderText">
       <string>Enter here...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>50</y>
        <width>71</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/image/image/send1.png);

color: rgb(255, 255, 255);
background-color: rgb(238, 240, 255, 140);

border:0px solid rgb(0, 0, 0);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>621</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(205,212,204, 255);

border-bottom-left-radius:0px;

border-bottom-right-radius:0px;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>0</y>
         <width>51</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-top-left-radius:10px;
image: url(:/image/image/pic3.png);


border-top-right-radius:10px;



border-bottom-left-radius:10px;

border-bottom-right-radius:10px;
color: rgb(255, 255, 255);

</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>0</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/image/image/flush4.png);
border-top-left-radius:10px;

border-top-right-radius:10px;



border-bottom-left-radius:10px;

border-bottom-right-radius:10px;

background-color: rgb(229, 231, 243);
color: rgb(255, 255, 255);
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>0</y>
         <width>51</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/image/image/file3.png);
border-top-left-radius:0px;

border-top-right-radius:0px;



border-bottom-left-radius:0px;

border-bottom-right-radius:0px;


color: rgb(255, 255, 255);
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>0</y>
         <width>51</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/image/image/time2.png);
border-top-left-radius:0px;

border-top-right-radius:0px;



border-bottom-left-radius:0px;

border-bottom-right-radius:0px;


color: rgb(255, 255, 255);
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <zorder>lineEdit</zorder>
     <zorder>pushButton</zorder>
     <zorder>groupBox_3</zorder>
     <zorder>messageWindow</zorder>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>340</y>
       <width>118</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>884</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
